.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [10.x, 12.x, 13.x, 14.x]
+        node-version: [12, 14, 16]
```

Comment:
What about v13 and v15? They are not LTS versions but we still officially support them so they should really be a part of our test matrix?

Reply:
But isn't v13 EOL?

Reply:
It is but I don't think there is good reason to further complicate the version support scheme unless it is really necessary. Simple min required version should be enough.

Reply:
We can assume that just a little segment of enthusiastic use non-LTS versions, also the probabilities to fail in v13 or v15 is too small. And we don't have the resources to maintain that kind of builds, and it's ok big project doesn't officially support non-LTS versions of Ubuntu (could work or not).